### PR TITLE
Fixes #1041: Adds twitter sharing option to settings

### LINKED ISSUE #1041: Add share to twitter
**Actual Behaviour**

share to fb is only there, share to twitter is missing.

**Expected Behaviour**

Twitter share should be there

**Steps to reproduce it**

Go to chat.susi.ai

**Would you like to work on the issue?**

Yes, working on it as part of gci

### PR DESCRIPTION
**Pull request by abishekvashok**
Fixes #1041

Changes:
Adds option for writing a tweet about susi AI on the settings page,
share section.

Demo Link: https://pr-1043-fossasia-susi-web-chat.surge.sh

Screenshots for the change: 
screenshot_20180113_171422
screenshot_20180113_171403

### CODE REVIEW

**Comment by abishekvashok** [comment]
Pls review.
Ping @fossasia/susi-ai-web-chat-admins

**Comment by abishekvashok** [comment]
@isuruAb done

**Comment by YashKumarVerma** [comment]
@isuruAb I'm working on adding google+ share. shall I make a pull request on @abishekvashok's Repo ?

**Comment by abishekvashok** [comment]
@YashKumarVerma after this gets merged, feel free to do so.

**Comment by YashKumarVerma** [comment]
sure @abishekvashok

File: .gitignore [hunk 1]
```diff
@@ -18,3 +18,4 @@ yarn-debug.log*
 yarn-error.log*
 jest
 ..env.swp
+package-lock.json
```
Comment (anshumanv) [inline .gitignore]: This was so needed. :+1:

Comment (abishekvashok) [reply .gitignore]: ;)

**Pull request merged**